docs/userguide/expectations.md

```diff
@@ -471,3 +471,55 @@ end;
 ```
 Since NULL is neither *true* nor *not true*, both expectations will report failure.
 
+# Provide a custom error message
```

Comment:
To be consistent with the rest of document please change that to: Adding custom failure message